.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
 		"no-tabs": 0,
 		"quotes": [2, "double", { "avoidEscape": true }],
 		"import/no-named-as-default": 0,
+		"semi": [2, "always"],
```

Comment:
What does the `2` mean in this context?

Reply:
2 is the error level, see: https://eslint.org/docs/user-guide/configuring#configuring-rules